pkgs/development/tools/build-managers/buildbot9/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv,
+  buildPythonApplication,
+  python,
+  fetchurl,
+  twisted,
+  jinja2,
+  sqlalchemy,
+  sqlalchemy_migrate,
+  zope_interface,
+  future,
+  dateutil,
+  autobahn,
+  plugins ? []
+}:
+
+buildPythonApplication (rec {
+  name = "buildbot-0.9.0rc1";
+  namePrefix = "";
+
+  src = fetchurl {
+    url = "https://pypi.python.org/packages/1b/d0/419ec2d6e0e75e4fbeb766d662dedfd9c1186b1c87b54d1f93d511abe78c/buildbot-0.9.0rc1.tar.gz";
```

Comment:
prefer the use of mirror instead of direct URL (there is plenty of examples in `pkgs/top-level/python-packages.nix`)